pallets/validators-manager/src/lib.rs

```diff
@@ -619,6 +630,58 @@ impl<T: Config> Pallet<T> {
         Ok(hex::encode(EthAbiHelper::generate_eth_abi_encoding_for_params_only(&eth_description)))
     }
 
+    /// This function generates the data needed to generate a confirmation for registering a new
+    /// collator. The implementation must match this schema:
+    /// https://github.com/Aventus-Network-Services/avn-bridge/blob/v1.1.0/contracts/AVNBridge.sol#L344-L345
+    pub fn convert_collator_registration_action_to_confirmation_signing_data(
+        action_id: &ActionId<T::AccountId>,
+        enable_data_logs: bool,
+    ) -> Result<String, DispatchError> {
+        let validators_action_data = Self::try_get_validators_action_data(action_id)?;
+        let mut activate_collator_params_concat: [u8; PACKED_ARGUMENTS_SIZE] = [0u8; 96];
```

Comment:
```suggestion
        let mut activate_collator_params_concat: [u8; PACKED_KEYS_SIZE] = [0u8; 96];
```
Is it possible to use the constant instead of `96` on the right-hand side as well?